server/models/social_account.ts

```diff
@@ -7,6 +7,7 @@ export type SocialAccountAttributes = {
   provider: string;
   provider_username: string;
   provider_userid: string;
+  user_id: string;
```

Comment:
We will need a migration to add this column.

Reply:
I think that this column already exists. It seems to be indexed on as well. Definitely let me know if I'm missing something though

Reply:
You're correct, it does already exist. Good catch.